zk.js/src/relayer.ts

```diff
@@ -83,15 +90,52 @@ export class Relayer {
     this.url = url;
   }
 
-  async sendTransactions(
-    instructions: any[],
-    _provider: Provider,
-  ): Promise<RelayerSendTransactionsResponse> {
+  /**
+   * Convenience function for sending and confirming instructions via Light RPC node.
+   * Routes instructions to Light RPC node and confirms the last transaction signature.
+   */
+  async sendAndConfirmSolanaInstructions(
+    ixs: TransactionInstruction[],
+    connection: Connection,
+    confirmOptions?: ConfirmOptions,
+    prioritizationFee?: PrioritizationFee,
```

Comment:
We should add the ``prioritizationFee`` to the relayer fee so that the relayer gets reimbursed for the cost.

Reply:
We could add the prioritization fee to the relayer as a property and add it when using getRelayerFee() and make the fee variable private to ensure that getRelayerFee() is being used everywhere.